tests/iio_common.c

```diff
@@ -126,6 +126,36 @@ struct iio_context * autodetect_context(bool rtn, const char * name, const char
 	return ctx;
 }
 
+/* How many input or output channels are in the device */
```

Comment:
That doesn't really seem to describe what this function does.

Reply:
yeah, sorry about that. dropped.